.chloggen/ocpdashboards.yaml

```diff
@@ -0,0 +1,35 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix OpenShift internal collector metrics dashboards to use `_total` suffix
+
+# One or more tracking issues related to the change
+issues: [3994]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: | 
+  The following settings (or missing metrics configuration) exposes metrics with `_total` suffix:
```

Comment:
@swiatekm @jaronoff97 do you know when this change was introduced?

Reply:
I cannot find it in the changelog 

* https://github.com/open-telemetry/opentelemetry-collector/releases
* https://github.com/open-telemetry/opentelemetry-collector-contrib/releases

Reply:
I didn't know about this neither. It should be documented as a breaking change, right?

Reply:
Yeah, this was detected by our upstream testing job with the monitoring test case. We are fixing the OCP dashboard with the PR but this will be a breaking change for users having custom dashboards. 

I'm testing the PR and will get back with the result.